src/app/pages/TopicPage/Pagination/index.jsx

```diff
@@ -191,29 +191,44 @@ const renderBlock = ({
 };
 /* eslint-enable react/prop-types */
 
+const getTranslations = translations => ({
+  pageXOfY: 'Page {x} of {y}',
+  previousPage: 'Previous Page',
+  nextPage: 'Next Page',
+  ...translations.pagination,
+});
+
 const Pagination = ({ activePage, pageCount }) => {
-  const { service } = useContext(ServiceContext);
+  const { service, translations } = useContext(ServiceContext);
   const blocks = buildBlocks(activePage, pageCount);
   if (!blocks) return null;
 
+  const { pageXOfY, previousPage, nextPage } = getTranslations(translations);
+  const summaryText = pageXOfY
+    .replace('{x}', `<b>${activePage}</b>`)
+    .replace('{y}', `<b>${pageCount}</b>`);
+
   const showLeftArrow = activePage > 1;
   const showRightArrow = activePage < pageCount;
 
   return (
     <Nav role="navigation" aria-label="Page" data-testid="topic-pagination">
-      {showLeftArrow && <LeftArrow activePage={activePage} />}
+      {showLeftArrow && (
+        <LeftArrow activePage={activePage}>{previousPage}</LeftArrow>
+      )}
       <TextSummary
         service={service}
         data-testid="topic-pagination-summary"
         // eslint-disable-next-line jsx-a11y/aria-role
         role="text"
-      >
-        Page <b>{activePage}</b> of <b>{pageCount}</b>
-      </TextSummary>
+        dangerouslySetInnerHTML={{ __html: summaryText }}
```

Comment:
Just curious why `dangerouslySetInnerHTML` for the summaryText is set within the `Nav` here?

Reply:
This is just the github diff being confusing 😕  it's actually on the `TextSummary` component

The split diff view makes it clearer what changed
![Screen Shot 2022-04-19 at 09 31 02](https://user-images.githubusercontent.com/99000097/163960138-9227297f-e13b-4368-a9ed-67fa79812f5a.png)